serverless-configs/aws-functions.yml

```diff
@@ -140,22 +140,22 @@
           rate: cron(0 12 ? * MON-FRI *)
           input:
             projection: 'epsg4326'
-          enabled: ${env:COLORMAP_JOB_ENABLED, false}
+          enabled: ${env:COLORMAP_JOB_ENABLED, true}
```

Comment:
My preference would be to leave these as false because if the variable is removed from bamboo it would still be set to true which I think is more confusing and harder to track

Reply:
I am in favor of it being true:
- The intended design is for these lambdas to run, so the default should be true. Anyone deploying EDSC needs these lambdas to run, or they'll get an error banner for every colormap loading attempt.
- The ability to disable the lambdas wasn't added until there was a need to disable colormap support for a while, but that was a temporary need we don't anticipate having again.
- The code is more accessible than Bamboo, so it's an easier place to check behavior. Generally I would like to see as few environment variables in routine use as possible.

Reply:
With these set to true does that mean that the jobs will run on these schedules in your dev environment? I think the answer to that is yes.

_If_ the answer is yes:
Is there any reason to have these jobs running locally on a schedule? No, and the less things you can have running locally the better
Can these jobs run in your local environment without errors? I think SQS is involved, so that would not work locally.

Reply:
I don't think they can run locally on a schedule without additional serverless plugins. So there's no risk of them running on a schedule locally.

Reply:
I switched the first schedule to this and set the default to true
```
rate: cron(0 * * * MON-FRI *)
```

and it ran with errors
```
Server ready: http://localhost:3001 🚀
Found 1086 layers within GIBS
sending message {"id":897,"product":"AMSR2_Cloud_Liquid_Water_Day","url":"https://gibs.earthdata.nasa.gov/colormaps/v1.3/AMSR2_Cloud_Liquid_Water_Day.xml"}
InvalidClientTokenId: The security token included in the request is invalid.
(λ: generateColorMaps) RequestId: 1117578b-2c9c-4a61-bf7d-46dac5a1731a  Duration: 1726.52 ms  Billed Duration: 1727 ms
Successfully invoked scheduled function: [generateColorMaps]
Completed Capabilities request with status 500.
```

Reply:
Thanks for the heads-up! I thought that they wouldn't run because I saw serverless plugins for that purpose, but apparently at some point after we disabled these lambdas in 2021, `serverless offline` added support for scheduled lambdas.

I just pushed a commit that prevents them from being scheduled in the local dev environment.

Reply:
I'm sure this solution can work but, it does seem really overcomplicated, having it just left as `false` seems like the correct solution here and consistent with the way the rest of our jobs worked